app/assets/javascripts/client/about/AboutSidebarCtrl.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('AboutSidebarCtrl', AboutSidebarCtrl);
+
+  AboutSidebarCtrl.$inject = [
+    '$scope',
+    '$rootScope',
+    '$element',
+    '$location',
+    '$timeout',
+    '$modal'
+  ];
+
+  function AboutSidebarCtrl($scope, $rootScope, $element, $location, $timeout, $modal) {
+    var vm = this;
+
+    new Clipboard("[data-clipboard-target]");
```

Comment:
Strange...I'm sure this is how Clipboard is initialized but it's a bit peculiar to me.